website/docs/guides/markdown-features/markdown-features-code-blocks.mdx

```diff
@@ -398,6 +398,61 @@ function MyPlayground(props) {
 
 </BrowserWindow>
 
+Or with [`import` statement syntax](https://github.com/nihgwu/react-runner#import-statement-and-multi-files)
+
+```jsx title="src/theme/ReactLiveScope/index.js"
+import React from 'react';
+import {createRequire} from 'react-runner';
+
+// highlight-start
+const ButtonExample = (props) => (
+  <button
+    {...props}
+    style={{
+      backgroundColor: 'white',
+      color: 'black',
+      border: 'solid red',
+      borderRadius: 20,
+      padding: 10,
+      cursor: 'pointer',
+      ...props.style,
+    }}
+  />
+);
+// highlight-end
+
+// Add react-live imports you need here
+const ReactLiveScope = {
+  React,
+  ...React,
+  // highlight-start
+  require: createRequire({
```

Comment:
This is technically a breaking change if this is required, right?

Reply:
As far as I understand your library is more powerful and allow to handle `import` in the live code block

Maybe for now, we could keep things simple, as it used to be, and just let users assume that the scope objects will be available without import? (like before)

<img width="999" alt="image" src="https://user-images.githubusercontent.com/749374/153245783-f241d76c-270a-47f2-a95a-3ec5471f0414.png">


Power users can still use more advanced APIs by reading your doc, we don't have to document everything about your lib ourselves.

Particularly, if this is supposed to be retro-compatible, let's first merge the lib change without docs + recommended API change, and once we are sure that your lib works fine for most users we can start documenting more advanced features. 

Remember this is a public API surface. We may not want to couple this extension point to a specific implementation lib. Former implementation did not import `react-live`, which is good because it now enables us to use your lib in a retrocompatible way. We'd rather also keep the opportunity to leave your lib in a retrocompatible way if we need to 🤪